### DISCUSSION TITLE
Suggestion: Allow post privacy to be restricted to members of a list

### DISCUSSION THREAD

**Discussion by SteveE1950** [mastodon/mastodon#20922] [General]
I'm a bit new to Mastodon, so I don't know if this idea runs counter to the overall Mastodon ethos.

Mastodon currently allows posts to be views as: public, unlisted, followers only, and mentioned people only.  It would be useful if the "mentioned people only" could be referenced via lists.  This would be easier than having to type the names of each person you want to view your post, especially when posts are frequently meant for a particular set of individuals.

For instance, family members, members of a club, or other subset of an individual's social circle.

**Comment by trwnh** [comment]
#5686
